src/_provisional.scss

```
@@ -158,4 +158,9 @@
 			color: #fff;
 		}
 	}
+
+	//This color will eventually be replaced by the new colors from the alert revamp PR.
+	&.icon-warning-light, .icon-warning-light {
+		color: #DF7200;
+	}
 }
```

While you're at it, the "parent" of the icon-warning-light should be `.provisional {` only instead of `.provisional, .experimental {` as we are moving away from .experimental in the future, like so:
```suggestion
}

.provisional {
	//This color will eventually be replaced by the new colors from the alert revamp PR.
	&.icon-warning-light, .icon-warning-light {
		color: #DF7200;
	}
}
```